test/util/GTestHelpers.h

```diff
@@ -134,15 +139,20 @@ class CopyShield {
 
  public:
   template <typename... Ts>
-  explicit CopyShield(Ts&&... args) requires(std::constructible_from<T, Ts...>)
+  requires std::constructible_from<T, Ts&&...> explicit CopyShield(Ts&&... args)
       : pointer_{std::make_shared<T>(AD_FWD(args)...)} {}
```

Comment:
Do we follow some kind of convention where the requires clause should sit in a declaration like this?

Reply:
I personally prefer them in the template declaration wherever possible, but to be honest, this was just because
I implemented those independently from you.